.eslintrc.json

```diff
@@ -0,0 +1,97 @@
+{
+    "extends": ["airbnb-base", "airbnb-typescript/base"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "./tsconfig.json"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
```

Comment:
Why do we need so many rules?

Reply:
The rules that say "error" can be deleted later, as they are enforced by airbnb already. 
The rules that are off is overriding the airbnb rules and turning them off.

Reply:
Why do we need to turn off some rules?

Reply:
These rules feels like stupid rules to me

- @typescript-eslint/return-await
- func-names
- function-paren-newline
- class-methods-use-this
- arrow-parens
- object-curly-newline
- no-plusplus
- import/order
- no-console
- no-bitwise
- no-mixed-operators
- no-await-in-loop
- no-continue
- no-labels
- no-restricted-globals
- no-restricted-syntax

These ones are controversial. Need some discussion:
- import/no-cycle
- no-param-reassign
- no-lonely-if
- new-cap

These ones I feel like should be turned into warnings:
- @typescript-eslint/no-use-before-define
- @typescript-eslint/no-unused-expressions
- @typescript-eslint/no-unused-vars

This one I forgot to fix:
- @typescript-eslint/lines-between-class-members

This one needs discussion:
- @typescript-eslint/type-annotation-spacing